controllers/app.controllers.js

```diff
@@ -0,0 +1,7 @@
+const { selectTopics } = require("../models/app.models")
+
+exports.getTopics = (req, res, next) => {
+    selectTopics().then((rows) => {
```

Comment:
Can you think of a clearer name than `rows` for this parameter?

Reply:
Changed it to topics.

Reply:
Or, hold on, as below, this could be 'result', to indicate the result of the request, and then we're sending it on a key which can be called topics.

Reply:
'Topics' seems fine. As having the name of the key changed breaks sending it on object - so this way we are getting the result of the query of topics and then rather than sending the whole thing on the response.body, we're putting the result(topics) on a key of topics. That makes sense to me.